backpack/__init__.py

```diff
@@ -160,10 +210,16 @@ def hook_run_extensions(module, g_inp, g_out):
         memory_cleanup(module)
 
 
-def run_extension_hook(module):
+def run_extension_hook(module: Module) -> None:
     """Execute the post extensions hook on a module after all BackPACK extensions.
 
     See the `post_backward_hook` argument of the `backpack` context manager for details.
+
+    Args:
+        module: current module
+
+    Raises:
+        RuntimeError: if any extension occurred during the backward hook
     """
     try:
         CTX.get_extension_hook()(module)
```

Comment:
This Error does not get thrown in tests anymore.
Maybe the whole method can be deleted/deprecated.
@f-dangel Discuss this in person.

Reply:
This was intended to provide an improved error message and can be removed. The method however cannot be removed.